### DISCUSSION TITLE
Magins on Paper type

### DISCUSSION THREAD

**Discussion by joaosilvagmr** [joaomlourenco/novathesis#231] [Q&A] [answered]
I am using the UMinho/EE template, however, when I set the media tipe for Paper, the margins are equal in each page.

The margins shouldn't be different in odd or even pages?

**Comment by b-pereira** [comment, marked as answer]
No, and that's the way it **has** to be. **ALL** the margins **have to have 2.5cm**: top, bottom, left and right. And yes, sounds counterintuitive if you're to print the document, but is as it is in the UM Formatting guidelines. If you're going to print the document, the stationary stores near _campi_ already have the _Ai_ file. So what you put in the digital version can differ from the printed one. **As the stationary stores' employees told me: give us the digital file and we do the rest. We are used to it.** By the way, if you're going to print, be careful with colouring and customizing. Each coloured page is somewhat expensive. 

Don't forget to mark as answered.
